src/Settings/ElevarSettings.php

```diff
@@ -21,8 +27,34 @@ public function description(): string
         return 'Interact with Elevar.';
     }
 
+    public function sections()
+    {
+        return [
+           [
+               'title' => 'Key',
+               'properties' => 'uuid',
+           ],
+           [
+               'title' => 'Data Layer Listener',
+               'properties' => 'data_layer_listener_enabled',
+           ],
+           [
+               'title' => 'Server Side',
+               'properties' => 'server_side_url',
+           ],
+        ];
+    }
+
     public static function group(): string
     {
         return 'elevar';
     }
+
+    protected function help()
+    {
+        return [
+            'data_layer_listener_enabled' => "Check to inject elevar code.",
+            'server_side_url' => 'Enter URL or Leave blank for null',
```

Comment:
The `data_layer_listener_enabled` is already pretty self explanatory, so you can omit that help.
I updated the language for `server_side_url` to avoid getting to technical with the use of "null" and leaking the code implementation into here.

```suggestion
            'server_side_url' => 'Enter server side URL if you have one.',
```